### PR TITLE
DBZ-663 disable cleaning of history topic based on size

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.jboss.org/browse/DBZ-663

### CODE REVIEW

File: debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java [hunk 1]
```diff
@@ -340,7 +340,7 @@ public void initializeStorage() {
 
             // Create topic
             final NewTopic topic = new NewTopic(topicName, (short)1, replicationFactor);
-            topic.configs(Collect.hashMapOf("cleanup.policy", "delete", "retention.ms", Long.toString(Long.MAX_VALUE)));
+            topic.configs(Collect.hashMapOf("cleanup.policy", "delete", "retention.ms", Long.toString(Long.MAX_VALUE), "retention.bytes", "-1"));
```
Comment (gunnarmorling) [inline debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java]: -1 is the default as per the [docs](https://kafka.apache.org/documentation/). Is this to protect against the case where a global default is given (if that's even doable)?

Comment (gunnarmorling) [reply debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java]: Ok, so IIUC, if there's a server default set for `retention.bytes`, it'll apply to this topic as we don't override the value when creating the topic. That's bad. In fact, I think this warrants an out-of-order release next Monday.

Comment (jpechane) [reply debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java]: Exactly, this is to cover the cases where default was overridden.

**Comment by gunnarmorling** [comment]
Applied. Thanks, @jpechane!

**Pull request merged**